### DISCUSSION TITLE
Error load certificate lets encrypt

### DISCUSSION THREAD

**Discussion by manustars** [0xJacky/nginx-ui#401] [Q&A]
I'm probably the one who isn't very capable. I generated a certificate with lets encrypt and so far everything is ok, I checked that the certificate and the key are actually in the ssl folder and I actually find the fullchain.cer and private.key
the certificate path is /home/manuel/NGINX/NGINX/ssl/\*.mydomain.com_mydomain.com_2048/ but I get this error
`2024/05/07 23:13:51 [emerg] 258#258: cannot load certificate "/home/manuel/NGINX/NGINX/ssl/\*.mydomain.com_mydomain.com_2048/fullchain.cer": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/home/manuel/NGINX/NGINX/ssl/\*.mydomain.com_mydomain.com_2048/fullchain.cer, r) error:10000080: BIO routines::no such file) nginx: [emerg] cannot load certificate "/home/manuel/NGINX/NGINX/ssl/\*.mydomain.com_marsellihome.com_2048/fullchain.cer": BIO_new_file() failed (SSL: error :80000002:system library::No such file or directory:calling fopen(/home/manuel/NGINX/NGINX/ssl/\*.mydomain.com_mydomain.com_2048/fullchain.cer, r) error:10000080:BIO routines:: no such file) nginx: configuration file /etc/nginx/nginx.conf test failed exit status 1`

for the path /etc/nginx/nginx.conf does not actually exist

this my docker-compose

version: '3.3'
services:
    nginx-ui:
        stdin_open: true
        tty: true
        container_name: nginx-ui
        restart: always
        environment:
            - TZ=Europe/Rome
        volumes:
            - '/home/manuel/NGINX/NGINX:/etc/nginx'
            - '/home/manuel/NGINX/NGINX-UI:/etc/nginx-ui'
            - '/var/www:/var/www'
        ports:
            - 8080:80
            - 8443:443
        image: 'uozi/nginx-ui:latest'

**Comment by 0xJacky** [comment]
Hi, @manustars can you provide the config content of this site? Thanks!

**Comment by 0xJacky** [comment]
Here is my configuration generated by nginx ui(v2.0.0-beta.23).
![image](https://github.com/0xJacky/nginx-ui/assets/13096985/403b197f-1b06-4a0c-9b88-2a12427d0614)

First, I obtained a wildcard certificate, then I create a site with TLS encrypted and select that certificate, everything seems to be working correctly.
